akka-docs/src/main/paradox/stream/operators/Source/completionStageSource.md

```diff
@@ -21,3 +36,14 @@ If the *completion* fails the stream is failed with that exception.
 
 @@@
 
+## Example
+
+Suppose we are accessing a remote service that streams user data over HTTP/2 or a WebSocket. We can model that 
+as a `Source<User,NotUsed>` but that source will only be available once the connection has established.
```

Comment:
```suggestion
as a @unidoc[Source[User,NotUsed]] but that source will only be available once the connection has been established.
```

Reply:
@raboof I've accepted this suggestion but nothing gets rendered now. I can't find another example usage of `@unidoc[...]` in the `akka/akka` docs either. Do we need a newer paradox?

Reply:
I've reverted while I discover what's the paradox-idiomatic alternative to back-ticks.